src/features/team-format-page.js

```diff
@@ -0,0 +1,132 @@
+        jquiJS  = $("<script>", {"src":"//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/jquery-ui.min.js", "type":"text/javascript"}),
+        teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+    
+    if(bttv.settings.get("darkenedMode") === true) {
+        teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page-dark.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+    }
+    $('body').append(jquiCSS, jquiJS, teamCSS);
+
+    $("h2").remove();
+    $("#about").prepend($("#banner_header"));
+
+    var followTeamButton = $("<a>", {"id":"followTeamBtn", "data-ember-action":"135", "class":"js-follow follow button primary action"});
+    followTeamButton.text("Follow The Whole Team");
+    followTeamButton.click(function(e) {
+        followTeam();
+    });
+    $("#team_logo").after(followTeamButton);
+
+    var newDiv = $("<div>", {"id":"team_chat"});
+    $(".wrapper.c12.clearfix").append(newDiv);
+
+    $(document).click(function(e) {
+        if(e.target.id =="sharebtn") {
+            var o = $(e.target).offset(),
+                offsetLeft = o.left + 1,
+                offsetTop = o.top + 42;
+            $("#share").css({"top":offsetTop+"px", "left":offsetLeft+"px"}).toggle("blind");
+        }
+    });
+
+    $(window).resize(function() {
+        var vpWidth = $(window).width(),
+            vpHeight = $(window).height(),
+            paddedWidth = vpWidth - 40;
+
+        if(paddedWidth < 985) {
+            paddedWidth = 985;
+        }
+
+        var playerWidth = paddedWidth - 685,
+            arMultiplier = (playerWidth - 2) / 16,
+            playerHeight = (9 * arMultiplier) + 32;
+
+        $("div.main.discovery.team").css("width", vpWidth+"px");
+        $("div.wrapper.c12.clearfix").css("width", paddedWidth+"px");
+        $("#player_column").css("width", playerWidth+"px");
+        $("#site_footer").css("width", playerWidth+"px");
+        $("#standard_holder").css({"width":(playerWidth - 2)+"px", "height":playerHeight+"px"});
+
+        if($("#team_chat").length) {
+            var chatHeight = $("#live_player").height() - 2;
+            $("#team_chat").css("height", chatHeight+"px");
+        }
+
+        var teamListHeight = ( ($("#live_player").height() - $("#team_logo").height()) - 76 );
+        $("#team_member_list").css({"height":teamListHeight+"px"});
+    });
+
+    var maxId = setTimeout(function() {}, 0);
+    for(var i=0; i < maxId; i+=1) { 
+            clearTimeout(i);
+    }
+
+    var followList = "none",
+        uName = cookie.get("login");
+        
+    vars.teamCurrentChannel = ($(".js-playing").attr("id")).replace("channel_", "");
+    vars.teamFirstLoad = 1;
+    setTimeout(loadTeam, 1000);
+    
+    var followTeam = function() {
```

Comment:
How you're following teams is not done well. Instead of recursively calling with a setTimeout, I'd suggest dumping channels to follow into a queue, and rolling that queue on an interval until the queue is empty. Thus, you avoid three separate functions. If a follow fails, you might just dump the user back into the queue (and keep track of errors).